terraform.tfvars

```diff
@@ -3,11 +3,11 @@ landing_zone_providers = {
     account_id = "123456789012"
     region     = "us-east-1"
   }
-  # ,
-  # another_account = {
-  #   account_id = "987654321098"
-  #   region     = "us-east-1"
-  # }
+
+#  sample = {
+#    account_id = "123456789015"
```

Comment:
```suggestion
#    account_id = "123456789012"
```

Reply:
well, this one I've missed. please change it from `123456789012` to `987654321098`